conda-recipe/meta.yaml

```diff
@@ -18,5 +18,5 @@ requirements:
   run:
     - python
     - pywin32
-    - pandas
+    - pandas >=0.17
```

Comment:
This ensures `sort_values` exists

Reply:
I was thinking about yesterday and I'm a bit torn. This is an API breaking change, but I got everything working with 0.19.2, which is compatible with python 3.4 and 3.6 Its still available in the API up to that point (though I'm not sure its the last version to have it).